packages/editor-preview/src/browser/editor-preview-widget.ts

```diff
@@ -1,5 +1,5 @@
 /********************************************************************************
- * Copyright (C) 2018 Google and others.
+ * Copyright (C) 2021 Ericsson and others.
```

Comment:
Same as earlier comment.

Reply:
changed to `2018-2021 Google and others`